extra/changelog-generator/changelog-generator

```diff
@@ -166,15 +167,20 @@ for repo in repos:
                                                  % sha)
                     exclusive_tag_seen = True
 
-                if re.match("^Title[ .]*$", match.group(1), re.IGNORECASE):
-                    log_entry = title
-                elif re.match("^Commit[ .]*$", match.group(1), re.IGNORECASE):
-                    log_entry_commit = True
-                elif re.match("^None[ .]*$", match.group(1), re.IGNORECASE):
-                    log_entry_commit = False
+                    if match_keyword.group(1).lower() == "Title".lower():
```

Comment:
The second `lower()` is very overkill, why not just type `title`? Same for the others.